src/games/strategy/triplea/delegate/MoveValidator.java

```diff
@@ -1675,17 +1699,18 @@ public static Route getBestRoute(final Territory start, final Territory end, fin
     if (!start.isWater() && !end.isWater()) {
       Route landRoute;
       if (isNeutralsImpassable) {
-        landRoute = data.getMap().getRoute_IgnoreEnd(start, end,
-            new CompositeMatchAnd<Territory>(Matches.TerritoryIsLand, noNeutral, noImpassible));
+        landRoute =
+            data.getMap().getRoute_IgnoreEnd(start, end,
+                new CompositeMatchAnd<Territory>(Matches.TerritoryIsLand, noNeutral, noImpassible));
       } else {
-        landRoute = data.getMap().getRoute_IgnoreEnd(start, end,
-            new CompositeMatchAnd<Territory>(Matches.TerritoryIsLand, noImpassible));
+        landRoute =
```

Comment:
Again, weird formatting, an accumulation of more technical debt. Unless you're formatting thousands and thousands of lines, the rule I know is you're not allowed to let the auto-formatter degrade the existing format. IE: boyscout rule, code is left as clean or cleaner than when you found it.